lib/experimental-default-theme.json

```diff
@@ -136,6 +136,9 @@
 			},
 			"fontSize": {
 				"custom": true
+			},
+			"lineHeight": {
+				"custom": false
```

Comment:
unlike the previous one, this one is opt-in and doesn't have presets yet.

Reply:
For context, there's https://github.com/WordPress/gutenberg/issues/23111 to create presets for line-height, padding, and margin that also lists previous experiments that were canceled.

Reply:
Perfect, it's a good way to test theme.json only presets.